src/CustomerioTracking.tsx

```diff
@@ -66,7 +73,18 @@ class CustomerIO {
       }
     }
 
-    CustomerioReactnative.initialize(env, config, packageConfig);
+    var writeKey = `${env.apiKey}:${env.apiKey}`; // For backward compatibility
+    if (env.writeKey && env.writeKey !== '') {
+      writeKey = env.writeKey;
+    }
+    console.log('Initialising using Segment');
```

Comment:
```suggestion
```

I am not sure if you intended to include this log statement in the PR or not.

Reply:
This is removed now.